pyflakes/checker.py

```diff
@@ -1216,6 +1226,10 @@ def handleNodeLoad(self, node):
                         scope[n.fullName].used = (self.scope, node)
                     except KeyError:
                         pass
+                if n.for_annotations and not self._in_annotation:
```

Comment:
Should this not happen before `scope[name].used` is marked? It's not really being used and the variable should be eligible to be released.

See: https://github.com/PyCQA/pyflakes/pull/622/files#diff-5df44d79406311387e0056faae7a8092d1ce9cc85c7d15420186e45a0c712283L1204-R1218

Reply:
If we do this, the following test will change:
```python
from typing import TYPE_CHECKING
if TYPE_CHECKING:                                                                                                                                                                              
    from a import b
b()
```
It will also return `'a.b' imported but unused` after this change. Perhaps that is better.

We can do this.

Reply:
Done. Thanks.